apps/codeforafrica/src/lib/api.ghost/posts.js

```diff
@@ -34,6 +32,26 @@ function transformPost(post) {
     month: "short",
     day: "numeric",
   });
+
+  const seo = {
+    title,
+    description: excerpt,
+    titleTemplate: `%s | ${primaryTag.slug} | CFA`,
```

Comment:
```suggestion
```

As we discussed, don't hard code. Create a new page, `Stories | Individual story` and allow users to set these values there.

Do similar thing for About pages i.e. `About | Partners`, `About | Partners | Individual partner`, `About | Members`, and `About | Members | Individual member`.